types/stylelint/index.d.ts

```diff
@@ -223,7 +223,7 @@ declare module 'stylelint' {
 			 */
 			cwd?: string;
 			ignoreDisables?: boolean;
-			ignorePath?: string;
+			ignorePath?: string[];
```

Comment:
[suggest] Since the `LinterOptions` type is a part of the public API, some plugins or API users' codes may break. Is it possible to keep backward compatibility like this?

```suggestion
			ignorePath?: string | string[];
```

Reply:
I fixed to allow string ignorePath. https://github.com/stylelint/stylelint/pull/6345/commits/450fe87f41a8bdd28eb1a0d60ba3064599881ca4